.circleci/config.yml

```diff
@@ -69,10 +52,14 @@ jobs:
           name: Setup e2e Environment
```

Comment:
Line 43 on the new file (restore_cache:) -- is that needed anymore since the save_cache above is removed?

Reply:
Yes, thank you.